src/chef_ci_status.rb

```diff
@@ -194,19 +335,152 @@ def print_pr_or_issue_stats(stats, item)
   puts "    Avg Time to Close #{item_plural}: #{avg_time_str}"
 end
 
-options = {
-  org: 'chef',
-  repo: 'chef',
-  branches: ['main'],
-  days: 30,
-  verbose: false,
-  mode: ['all'],
-}
+# Define defaults from config or use hardcoded values
+options = if defined?(Settings)
+            default_mode = if Settings.default_mode.is_a?(Array)
+                             Settings.default_mode
+                           else
+                             [Settings.default_mode]
+                           end
+            # Get CI timeout from config with fallback to default
+            # Get timeout with fallback
+            has_timeout = Settings.respond_to?(:ci_timeout)
+            ci_timeout = has_timeout ? Settings.ci_timeout : 180
```

Comment:
if we're gonna have a default, shouldn't we fill that in in the Settings module?